### PR TITLE
feat(EMS-3975): broker manual address - initial page setup

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
As part of "Broker - address lookup" functionality, there is a new "Enter address manually" page.

This PR introduces the initial setup for this page.

## Resolution :heavy_check_mark:
- Create new `BROKER_MANUAL_ADDRESS` route constant.
- Create new page content strings.
- Create new E2E test.
- Create new nunjucks template.
- Create new GET route and controller.
- Address TODO comments.
- Fix a failing date test.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/policy/broker-manual-address/broker-manual-address-page.spec.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const {
+  ROOT,
+  POLICY: { BROKER_MANUAL_ADDRESS_ROOT },
+} = INSURANCE_ROUTES;
+
+const { BROKER_MANUAL_ADDRESS: FIELD_STRINGS } = FIELDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Policy - Broker manual address page - As an exporter, ... TODO', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/policy/broker-manual-address/broker-manual-address-page.spec.js:24]: TODO comment?

Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/policy/broker-manual-address/broker-manual-address-page.spec.js:24]: Thanks, we didn't have a fully populated ticket, we do now. Updated.

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js [hunk 2]
```diff
@@ -100,12 +100,6 @@ context('Insurance - Change your answers - Policy - Broker - Yes to no - As an e
 
         cy.assertEmptyFieldValue(NAME);
         cy.assertEmptyFieldValue(EMAIL);
-
-        // TODO: EMS-3975
-        // cy.checkTextareaValue({
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js:105]: will this not need to be tested in a future ticket?

Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js:105]: Yes, when the flow is changed and the check/change answers flow is updated. ~3-5 PRs are required for that.

So just removed it from this test to save having commented code floating around.

File: e2e-tests/insurance/cypress/e2e/journeys/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js [hunk 3]
```diff
@@ -82,12 +82,6 @@ context('Insurance - Policy - Change your answers - Broker - Yes to no - As an e
 
         cy.assertEmptyFieldValue(NAME);
         cy.assertEmptyFieldValue(EMAIL);
-
-        // TODO: EMS-3975
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js:86]: Replied above.

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js (hunk 2 above)
Comment (Zainzzkk) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js:105]: great, as long as we remember to add the test

**Review by Zainzzkk — APPROVED** [review]

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js (hunk 2 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/broker/change-your-answers-change-broker-yes-to-no.spec.js:105]: We will have to, to test changing broker from yes to no and vice versa, once the full flow is in place

**Review by Zainzzkk — APPROVED** [review]

**Review by Zainzzkk — APPROVED** [review]

**Review by Zainzzkk — DISMISSED** [review]

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**